external/fv3fit/fv3fit/train_microphysics.py

```diff
@@ -234,7 +237,13 @@ def open_dataset(
         self, url: str, nfiles: Optional[int], required_variables: Set[str],
     ) -> tf.data.Dataset:
         nc_open_fn = self.transform.get_pipeline(required_variables)
-        return nc_dir_to_tf_dataset(url, nc_open_fn, nfiles=nfiles)
+        return nc_dir_to_tf_dataset(
+            url,
+            nc_open_fn,
+            nfiles=nfiles,
+            shuffle=True,
```

Comment:
Why shuffle?

Reply:
We weren't shuffling before, so only Jan data was used to fit all the normalization. It's possible this could have an impact since it is southern summer, and the coldest temperatures are not present.